common/src/main/java/org/figuramc/figura/lua/api/ClientAPI.java

```diff
@@ -634,6 +638,28 @@ public static Map<String, Map<String, Object>> getScoreboard() {
         return map;
     }
 
+    @LuaWhitelist
+    @LuaMethodDoc("client.get_bossbars")
+    public static Map<String, Object> getBossbars() {
+        BossHealthOverlay bossBars = Minecraft.getInstance().gui.getBossOverlay();
+        Map<String, Object> bosses = new HashMap<String, Object>();
+        for (Map.Entry<UUID, LerpingBossEvent> entry : ((BossHealthOverlayAccessor) bossBars).getBossEvents().entrySet()) {
+            Map<String, Object> contents = new HashMap<String, Object>();
+
+            BossEvent event = entry.getValue();
+            contents.put("name",event.getName());
+            contents.put("progress",event.getProgress());
+            contents.put("color",event.getColor().getName());
+            contents.put("style",event.getOverlay().getName());
+            contents.put("darkenscreen", event.shouldDarkenScreen());
+            contents.put("bossmusic",event.shouldPlayBossMusic());
```

Comment:
thou shall camel-case
```suggestion
            contents.put("darkenScreen", event.shouldDarkenScreen());
            contents.put("bossMusic",event.shouldPlayBossMusic());
```